gnat2goto/driver/tree_walk.adb

```diff
@@ -3546,6 +3604,14 @@ package body Tree_Walk is
                      I_Type          => Ret_Type);
    end Do_Operator_Sub_Mod;
 
+   function Do_Operator_Sub_Mod (N : Node_Id) return Irep is
```

Comment:
I guess you could squash `do_operator_mod` and `do_operator_sub_mod` into one function. But that's just a suggestion.